vars/republishDeployPipeline.groovy

```diff
@@ -26,59 +26,73 @@ def call(config) {
     println(params.changeLog)
   }
 
-  def namespacesTestParsed = parseJson(config.namespacesTest)
-  if (namespacesTestParsed != null && namespacesTestParsed.size() > 0) {
+  if (config.namespacesTest != null && config.namespacesTest.size() > 0) {
     cdNode {
-      namespacesTestParsed.each { item ->
-        stage("Helm Deploy: ${item.namespace}") {
-          def chartEnv = item.chartOverride
-          if (!chartEnv) {
-            chartEnv = item.namespace
-          }
-          withEnv(["CHART_ENVIRONMENT=${chartEnv}"]) {
-            helmDeploy([
-                    releaseName          : config.application,
-                    namespace            : item.namespace,
-                    imageTag             : params.imageTag,
-                    helmfileRepoOverride : config.helmfileRepoOverride
-            ])
-          }
+      for (namespaceTest in config.namespacesTest) {
```

Comment:
This doesn't seem correct? This section should handle deploying the regular application if I am not mistaken, the testing is handled below?

Reply:
This is reverted to how it previously was.  One can add test namespaces that would be deployed before integ-test and pre-prod namespaces.

We can also remove this entirely?

Reply:
@JacusSH I see what you mean, was confused by the diff but looking at the whole file it makes sense now. I think we can probably just remove namespacesTest entirely to be honest, we don't have much use-case for it